code/modules/admin/verbs/onlyone.dm

```diff
@@ -3,33 +3,70 @@
 		alert("The game hasn't started yet!")
 		return
 
-	for(var/mob/living/carbon/human/H in player_list)
-		if(H.stat == 2 || !(H.client)) continue
-		if(is_special_character(H)) continue
-
-		ticker.mode.traitors += H.mind
-		H.mind.special_role = "traitor"
+	switch(alert("Delayed or no?", "PICK ONE!", "NOW!", "Delayed", "Cancel"))
+		if("Cancel")
+			return
+		if("Delayed")
+			for(var/M in player_list)
+				playsound(M, "sound/misc/highlander_delayed.ogg", 40)
+				to_chat(world, "<font size = 4 color=red>Bagpipes begin to blare. You feel Scottish pride coming over you.</span></font>")
```

Comment:
This is not the only instance that spams one message for every mob hearing it:
https://i.gyazo.com/32e0b63928e00be2d14e795a9fed3118.png
But it's a good example.